srcpkgs/gutenprint/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'gutenprint'
 pkgname=gutenprint
-version=5.2.14
+version=5.3.1
 revision=1
 build_style=gnu-configure
 hostmakedepends="ghostscript perl"
 makedepends="cups-devel"
 short_desc="Top quality printer drivers for POSIX systems"
 maintainer="Anachron <Anachron14@gmx.de>"
-license="GPL-2"
+license="GPL-2.0-or-later"
 homepage="http://gimp-print.sourceforge.net/"
-distfiles="${SOURCEFORGE_SITE}/gimp-print/$pkgname-$version.tar.bz2"
-checksum=45bf0766f196a764c113bfad7ab1ea748f1d5ff9ebb0d3a2206705fe52e82389
+distfiles="${SOURCEFORGE_SITE}/gimp-print/$pkgname-$version.tar.xz"
+checksum=d80d8f5272d15bca5710f6ef4a2bb95e76d34e1155c5bcd5e83cb9f0a111d0d4
 configure_args="--disable-rpath \
```

Comment:
this configure_args is qack, can you please put it properly directly under build_style ?